Folder/email_notifier.py

```diff
@@ -0,0 +1,32 @@
+import os
+import smtplib
+import requests
+from dotenv import load_dotenv
+
+
+class EmailNotifier:
+
+    def __init__(self):
+        self._smtp_username = os.environ["SMTP_USERNAME"]
+        self._smtp_password = os.environ["SMTP_PASSWORD"]
+        self._sheety_users_endpoint = os.environ["SHEETY_USERS_EP"]
```

Comment:
nitpick butr you might wanna rename `SHEETY_USERS_EP` to `SHEETY_USERS_ENDPOINT`. Unless you can later remember EP stands for endpoint, which isn't a common abbreviation